Sources/KarrotListKit/List.swift

```diff
@@ -55,6 +55,24 @@ extension List {
     registerEvent(PullToRefreshEvent(handler: handler))
   }
 
+  /// Register a callback handler that will be called when the user scrolls near the end of the content view.
+  ///
+  /// - Parameters:
+  ///   - offset: The offset from the end of the content view that triggers the event. Default is two times the height of the content view.
+  ///   - asyncAction: The callback handler that performs an asynchronous action when the end is reached.
+  /// - Returns: An updated `List` with the registered event handler.
+  public func onReachedEnd(
```

Comment:
`List`의 기존 `onRefresh`나, 기타 SwiftUI 모디파이어처럼 `on` + `동사원형` 형태의 네이밍으로 가는 건 어떨지 제안드려요!

```suggestion
  public func onReachEnd(
```

Reply:
좋네요! 기존의 `ReachedEndEvent`도 ReachEndEvent로 변경하는 것이 좋으려나요? 🤔
영어 문법이 어렵네요..

Reply:
생각해보니 `on `이 전치사로 붙었을대 동사원형을 쓰는 것이니 `ReachedEndEvent` 요건 기존을 유지하는 것이 맞겠네요!

Reply:
[1bea6e0](https://github.com/daangn/KarrotListKit/pull/44/commits/1bea6e0ba5afc1d5fbbc2dc98737ef1711b6c643) 에서 반영했어요.